docs/05.PORT-API.md

```diff
@@ -0,0 +1,287 @@
+# Reference
+
+## Termination
+
+It is questionable whether a library should be able to terminate an application. However, as of now, we only have the concept of completion code around `jerry_parse` and `jerry_run`. Most of the other API functions have no way of signaling an error. So, we keep the termination approach with this port function.
```

Comment:
Good question. AFAIK, "out of memory" errors can happen in `jerry_create_...` calls. @zherczeg ?